bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/seq/ItcTable.scala

```diff
@@ -166,6 +166,11 @@ trait ItcTable extends Table {
 
   }
 
+  // lacking a simple way to decide on the "closest" (i.e. "fastest to reach") peer we talk to the observing peer
+  // if defined and GN or GS otherwise, if no site is defined you're out of luck and the ITC tables will stay empty
+  protected def itcServicePeer: Option[Peer] =
+    ObservingPeer.get orElse ObservingPeer.peerFor(Site.GN) orElse ObservingPeer.peerFor(Site.GS) orElse None
```

Comment:
Yes, this is the service peer, so to say. I *think* defaulting to GN is what we're doing here by calling ```ObservingPeer.peerFor(Site.GN)``` ? So if that doesn't work (and there also is no peer defined for GS), I don't think there's anything else we can do.